src/js/util/format.js

```diff
@@ -538,8 +538,8 @@ const formatPlaylist = function (data) {
     playlist.followers = data.followers.total;
   }
 
-  if (data.tracks && data.tracks.total !== undefined) {
-    playlist.tracks_total = data.tracks.total;
+  if (data.tracks && data.tracks_total !== undefined) {
+    playlist.tracks_total = data.tracks_total;
```

Comment:
From memory, this code was to pluck the `total` sub-property from the `tracks` object, which if provided, should trump the value of `tracks.length`. Most providers paginate their APIs, so the `tracks.length` would typically be that page, rather than the total count.

I recommend undoing this change unless there is a specific bug that relates to this.